docs/content/zh/monitor/apply_monitoring.md

```diff
@@ -0,0 +1,139 @@
+# 开启监控
+
+## 简介
+
+[Prometheus](https://prometheus.io/) 基于文本的暴露格式，已经成为云原生监控领域事实上的标准格式。
+
+RadonDB MySQL 监控引擎基于 [Prometheus MySQLd Exporter](https://github.com/prometheus/mysqld_exporter) 定义。通过 `mysqld-exporter` 抓取 RadonDB MySQL 服务指标，再通过接入第三方应用平台实现监控指标可视化。
+
+本教程演示如何开启 RadonDB MySQL 监控指标。
+
+## 约束限制
+
+- RadonDB MySQL Operator 2.1.0及以上版本，支持自动开启监控引擎。
+
+## 准备工作
+
+- 已准备可用 Kubernetes 或 KubeSphere 集群。
+
+## 部署步骤
+
+### 步骤 1: 配置 serviceMonitor
+
+`serviceMonitor` 是定义 RadonDB MySQL Operator 自动监控引擎的参数，开启后将自动绑定 `mysqld_exporter` 与 Prometheus。
+
+`serviceMonitor`定义包含如下字段：
+
+```bash
+serviceMonitor:
+  enabled: true
+  ## Additional labels for the serviceMonitor. Useful if you have multiple prometheus operators running to select only specific ServiceMonitors
+  # additionalLabels:
+  #   prometheus: prom-internal
+  interval: 10s
+  scrapeTimeout: 3s
+  # jobLabel:
+  # targetLabels:
+  # podTargetLabels:
+  namespaceSelector:
+    any: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/managed-by: mysql.radondb.com
+      app.kubernetes.io/name: mysql
+```
+
+您可以在 `charts/mysql-operator/values.yaml` 文件中配置`serviceMonitor`。
+
+- 新部署 Operator 时， `serviceMonitor.enabled` 默认为**true**，表示默认开启。
+- 已部署 Operator 的集群，需重新部署 Operator。
```

Comment:
The deployed cluster of the **old version** of the operator needs to be redeployed.

Reply:
done